src/Serilog.Sinks.ElmahIO/Sinks/ElmahIo/ElmahIoSinkOptions.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Serilog.Core;
+using Serilog.Events;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Serilog.Sinks.ElmahIo
+{
+    /// <summary>
+    /// Provides ElmahIoSink with configurable options
+    /// </summary>
+    public class ElmahIoSinkOptions
+    {
+        /// <summary>
+        /// 
```

Comment:
Could use some comments as they do appear in the intellisense?

Reply:
Indeed. Documentation will be added once everything is code complete. But good point 👍